src/org/traccar/database/ConnectionManager.java

```diff
@@ -40,13 +40,15 @@
     private static final long DEFAULT_TIMEOUT = 600;
 
     private final long deviceTimeout;
+    private final boolean statusHandler;
```

Comment:
I think it's better to rename variable to something like `enableStatusEvents`. We can keep config key the same for consistency with others, but it's better to change variable name to make source code more clear.